py3status/modules/fedora_updates.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+"""
+Displays the number of package updates pending for a Fedora Linux installation.
+This will display a count of how many `dnf` updates are waiting
+to be installed.
+Additionally check if any update security notices.
+
+Configuration parameters:
+    - cache_timeout : how often we refresh this module in seconds
+      default is 600
+    - format : display format
+      default is 'DNF: {updates}'
+    - color_good : color when no upgrades needed
+      default is global color_good
+    - color_degraded : color when upgrade available
+      default is global color_degraded
+    - color_bad : color when security notice
+      default is global color_bad
+
+Format status string parameters:
+    - updates : number of pending dnf updates
+
+@author Toby Dacre <https://github.com/tobes>
+@license BSD
+"""
+
+from time import time
+import subprocess
+import re
+
+
+class Py3status:
+    # available configuration parameters
+    cache_timeout = 600
+    format = 'DNF: {updates}'
+    color_good = None
+    color_bad = None
+    color_degraded = None
+
+    _reg_ex_sec = re.compile('\d+(?=\s+Security)')
```

Comment:
those non configuration parameters may be better in a proper __init__(self) function please